ci/install.sh

```diff
@@ -45,11 +45,23 @@ EOF
     fi
 }
 
+install_zsh() {
+    case $TARGET in
+        aarch64-unknown-linux-gnu)
```

Comment:
Yeah, i kind of just copy-and-pasted the section above it without thinking too much about it. In hind-sight i see that it special-cased ARM because the C tool chain is arch-specific. But since you mentioned you didn't write these scripts i assume even that is probably just boiler-plate.